tests/devices/test_default_qubit.py

```diff
@@ -2054,7 +2054,7 @@ def test_s_inverse():
         test_s_inverse()
         operations = test_s_inverse.qtape.operations
         assert "S.inv" not in [i.name for i in operations]
-        assert "PhaseShift.inv" in [i.name for i in operations]
+        assert "PhaseShift" in [i.name for i in operations]
```

Comment:
With this change, `qml.inv()` now uses `qml.adjoint()` internally. As a result. gates that PL knows how to invert are automatically inverted; `PhaseShift(x)` becomes `PhaseShift(-x)` (rather than `PhaseShift(x).inv`).

The only exception are gates where the inverse is not known.

Reply:
Oh! That makes sense. thank you! :slightly_smiling_face: